doc/env_vars.md

```diff
@@ -6,64 +6,65 @@ All configuration variables can either be set directly via the command line as e
+| `LRSQL_LOG_LEVEL` | `logLevel` | The logging level to use. Can be `ALL`, `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR` or `OFF` | `INFO`  |
 
 ### Database
 
-| Env Var | Config | Description | Default |
-| --- | --- | --- | --- |
-| `LRSQL_DB_TYPE` | `dbType` | The DBMS that lrsql will use. Currently supports `h2:mem`, `h2`, `sqlite`, and `postgres`/`postgresql`. The default value used depends on the `main` entry point used, e.g. `lrsql.sqlite.main` will use `sqlite` by default, so overriding the default is **not** recommended. | Varies |
-| `LRSQL_DB_NAME` | `dbName` | The name of the database. | `example` |
-| `LRSQL_DB_HOST` | `dbHost` | The host that the database will run on. Not supported by in-mem H2 or SQLite. | `0.0.0.0` |
-| `LRSQL_DB_PORT` | `dbPort` | The port that the database will run on. Not supported by in-mem H2 or SQLite. | `9001` (H2), `5432` (PG) |
-| `LRSQL_DB_PROPERTIES` | `dbProperties` | Optional additional DB properties. Must be a string of key-val pairs that follow URL query parameter syntax (e.g. `key1=value1&key2=value2`). Any `&` character not used as a separator must be percent encoded. Supported properties will depend on the DBMS. | Not set |
-| `LRSQL_DB_JDBC_URL` | `dbJdbcUrl` | Optional JDBC URL; this will override the above properties if set. URL syntax will depend on the DBMS. | Not set |
-| `LRSQL_DB_USER` | `dbUser` | The DB user. Optional. | Not set |
-| `LRSQL_DB_PASSWORD` | `dbPassword` | The DB password. Optional. | Not set |
-| `LRSQL_DB_SCHEMA` | `dbSchema` | The DB schema. Optional. | Not set |
-| `LRSQL_DB_CATALOG` | `dbCatalog` | The DB catalog. Optional. | Not set |
+| Env Var               | Config         | Description                                                                                                                                                                                                                                                     | Default     |
+| --------------------- | -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `LRSQL_DB_TYPE`       | `dbType`       | The DBMS that lrsql will use. Currently supports `sqlite`, and `postgres`/`postgresql`. The default value used depends on the `main` entry point used, e.g. `lrsql.sqlite.main` will use `sqlite` by default, so overriding the default is **not** recommended. | Varies      |
+| `LRSQL_DB_NAME`       | `dbName`       | The name of the database.                                                                                                                                                                                                                                       | `example`   |
```

Comment:
Should mention that this var should be set to `:memory:` if we want SQLite in-mem mode.